x-pack/plugins/endpoint/server/routes/metadata/index.ts

```diff
@@ -125,3 +125,10 @@ function mapToHostResultList(
     };
   }
 }
+
+function enrichHostMetadata(hostMetadata: HostMetadata): HostInfo {
+  return {
+    metadata: hostMetadata,
+    host_status: HostStatus.ERROR,
```

Comment:
This will change once we have the actual host_status from the the ingest manager right?

Reply:
> This will change once we have the actual host_status from the the ingest manager right?

Yes, sir. And it will serve as a port into enrichment services and will become async eventually.